doc/source/whatsnew/v0.21.0.txt

```diff
@@ -155,7 +155,7 @@ Indexing
 
 - When called with a null slice (e.g. ``df.iloc[:]``), the ``.iloc`` and ``.loc`` indexers return a shallow copy of the original object. Previously they returned the original object. (:issue:`13873`).
 - When called on an unsorted ``MultiIndex``, the ``loc`` indexer now will raise ``UnsortedIndexError`` only if proper slicing is used on non-sorted levels (:issue:`16734`).
-
+- Fixes regression in 0.20.3 that raises an index error when a string is used for ``TimedeltaIndex`` (:issue:`16896`).
```

Comment:
can you be a bit more explicty (e.g. say indexing with a string  when you have a TimedeltaIndex)